themes/default/content/blog/generative-ai-apps-devops-talks-pulumi-user-group/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Unlocking the Benefits of LangChain AI for Dev, Sec and Ops"
+date: 2023-10-26
+meta_desc: "Learn LLMs and GenAI using LangChain AI, with key lessons for DevOps broken down into Development, Operations, and Security."
+meta_image: langchain-ai-gen-ai-workshop-llm-devops.png
+authors:
+    - sara-huddleston
+    - kat-morgan
+tags:
+   - pulumi-events
+   - community
+   - ai
+   - devops
+   - langchain
+---
+
+The emergence of DevOps revolutionized software development. Now, with AI powered tools like LangChain, these transformations are being accelerated. Unsurprisingly, our distinguished speaker at the launch of Pulumi's in-person AI Talks, Patrick Debois, who coined the term "DevOps," has recently tuned into LLM and GenAI Ops using the Langchain framework.
+
+This article unwraps the topics addressed during our "Generative AI, Apps, and DevOps | AI/ML Talks" event in Seattle, focusing on Langchain and GenAI for DevOps. You'll be able to [watch Patrick Debois teach LangChain to learn LLMs and GenAI](#how-to-use-langchain-to-learn-llms-and-genai-for-dev-sec-ops), with code examples and lessons that are easily understood by traditional software engineering.
```

Comment:
The link doesn't go anywhere or do anything, is it supposed to?

Reply:
I was trying to make it go to the section on the article itself, but I guess it didn't work. I don't know how to fix it.